tests/Language/SchemaPrinterTest.php

```diff
@@ -63,9 +63,13 @@ public function testPrintsKitchenSink(): void
 """
 type Foo implements Bar & Baz & Two {
   one: Type
-  """This is a description of the `two` field."""
+  """
+  This is a description of the `two` field.
+  """
```

Comment:
Why are those not one-liners with `"`?

Reply:
I tried to change as little as possible. Here it was not a one-liner in `schema-kitchen-sink.graphql` fixture.

Reply:
Got it. How would a description like the following be transformed:

```
"""one liner in multi quotes"""
type Foo
```

I guess it would not stay the same, but rather switch to either a one liner with single quotes or a block with triple quotes.

Reply:
Yes, there are two options according to spec

```
"one liner in multi quotes"
type Foo
```

```
"""
one liner in multi quotes
"""
type Foo
```

Reply:
I think we should go with option one and add a test to ensure consistent behaviour.

Reply:
What do you mean 🤔 Both are possible, depends on the user what he defines.

I can adapt our tests (fixtures) to always use single quotes.

When you set `description` with single line value with no trailing quote or slash, `"` is used.

Reply:
Please re-read https://github.com/webonyx/graphql-php/pull/1188#discussion_r908691474.

The example would be a user-defined schema. I am assuming that during parsing some information will be lost, so when the schema is then printed it will not output exactly the same.

Reply:
Ah now I get the initial question.

```gql
"""one liner in multi quotes"""
type Foo
```
is parsed as a BlockString token since our parser cares about triple quotes.

Reply:
Fine, that would mean the printed output would be option 2. That is fine with me, but we should cover this behaviour with a test.

Reply:
So schema printer test now covers it

https://github.com/webonyx/graphql-php/pull/1188/files#diff-61fa379df85242a4f4a4f5029dc7fde8f6d8bd0c3989e50124aa9421de9644d5R17